modules/nextflow/src/main/groovy/nextflow/script/ScriptRunner.groovy

```diff
@@ -226,6 +227,8 @@ class ScriptRunner {
     protected void parseScript( ScriptFile scriptFile, String entryName ) {
         scriptParser = new ScriptParser(session)
                             .setEntryName(entryName)
+                            // setting module true when running in "inspect" mode to prevent the running the entry workflow
+                            .setModule(ContainerInspectMode.active())
```

Comment:
This will still run the entry workflow if `concretize` is enabled, is that intended?

Reply:
Good point, initially I was thinking the run was needed by the contretize to await for the container completion, but checking it now it implemented [here](https://github.com/nextflow-io/nextflow/blob/0aa76affe22c35fc5bc6a4b609fa9b78528859a9/modules/nextflow/src/main/groovy/nextflow/container/inspect/ContainersInspector.groovy#L101-L101). 

therefore the "breaking" change is needed the `ContainerInspectMode` class

Reply:
Solved by https://github.com/nextflow-io/nextflow/pull/5605/files